docs/horizen_eon/07-tutorials/01-forger_node_setup_guide.md

```diff
@@ -330,7 +334,43 @@ docker compose -f deployments/forger/eon/docker-compose.yml exec evmapp gosu use
+
+A new instance of the smart contract is required for each forger: if you want to deploy one for your forger, the factory is available at this address: [0x8604Bb903B7D54F666bA1e75f98045345C63132a ](https://eon-explorer.horizenlabs.io/address/0x8604Bb903B7D54F666bA1e75f98045345C63132a?tab=contract)
+
+You can call the method deployDelegatedStakingReferenceImplementation of the factory, parameters required are the signPubKey and vrfKey that identify your forger (the latter is split in two different parameters: one for the first 32bytes and one for the last btye).
+The method execution will trigger the deploy of the smart contract instance.  Take note of its address: you will need it in the following setp.
+
+**(Note: this step is not needed if in the  following forger registration step you specify rewardShare = 0, meaning you don't wont to split the rewards and send all of them to the forger)**
+
+
+### Forger registration
+Starting from EON 1.4, a forger on-chain registration is required before being able to accept delegations and forge blocks.
+You can launch the transaction using the http endpoint  [/transaction/registerForger](https://github.com/HorizenOfficial/eon/blob/main/doc/api/transaction/registerForger.md)  in your local node.
+The mandatory parameters are:
+- blockSignPubKey and vrfPubKey: use the keys generated in the previous 'Generate keys' point
+- stakedAmount: the initial stake you want to assign to your forger. Represented as an integer value specified in zennies, and must be >= 10 ZEN. The amount will be taken from your local node wallet, and you will be able to withdraw it any time using the same address used by sending this transaction (but keep in mind that if the total delegated amount will become < 10 ZEN, your forger will no more be able to propose blocks).
+- rewardShare:  Reward to be redirected to a separate reward address (integer, range from 0 to 1000 - where 1000 represents 100%)
+- rewardAddress: External reward address (may be a single EOA or (more likely) a smart contract handling rewards distribution to delegator - see previous point). Must be present only if rewardShare is > 0. Omit the initial 0x prefix when specifing it.
+
+**Important:**<br>
+Double check rewardShare and rewardAddress parameters before launching the transaction, since they will not be updatable once set! If you want to change them after the registration, 
+you will have to register a new forger with different keys, and move all the delegations to the new forger.
+
+Check the [/transaction/registerForger](https://github.com/HorizenOfficial/eon/blob/main/doc/api/transaction/registerForger.md) EON documentation page for further info on this endpoint.
+
+Existing forgers present before eon 1.4 will not have to perform this step, and will be registered by default, with rewardShare = 0.
 
 ### Stake $ZEN to your Forger
 **Prerequisites**
```

Comment:
typo:

- at line 393: **Before proceeding => it misses closing ** so it is shown as "**Before proceeding"